content/en/docs/external-add-ons/bentoml/introduction.md

```diff
@@ -1,19 +1,44 @@
 +++
-title = "BentoML"
-description = "Build, Ship, and Scale AI Applications"
-weight = 45
-
+title = "Introduction"
+description = "A brief introduction to BentoML Yatai"
+weight = 1
```

Comment:
I think, we need review from BentoML folks for these changes.
/assign @yubozhao @ssheng